src/components/Navbar/Navbar.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Navbar from './Navbar';
+
+describe('Navbar component', () => {
+  const renderNavbar = ():void => {
+    render(
+      <Router>
+        <Navbar />
+      </Router>
+    );
+  };
+
+  it('renders Navbar component', async () => {
+    renderNavbar();
+
+    const linkHome = screen.getByRole('link', { name: /Home/i });
+    expect(linkHome).toBeInTheDocument();
+
+    const linkAbout = screen.getByRole('link', { name: /About/i });
+    expect(linkAbout).toBeInTheDocument();
+  });
+
+  it('check if the active class is applied correctly', async () => {
+    renderNavbar();
```

Comment:
you also can do like that:
```
const { getByRole, getByTestId... } = renderNavbar();
expect(getByRole('link', { name: /Home/i })).toBeInTheDocument();
```

Reply:
I cant use it in my situation, because i have an error 
![image](https://github.com/DGMary/quiz/assets/25418241/55dc1c2e-2a15-42e8-b48e-89cdba88bf9c)